modules/banner/css/lib/variables.scss

```diff
@@ -1,17 +1,18 @@
-$wdc-banner-border-radius: 3px;
+$wdc-banner-border-radius: 4px;
 $wdc-banner-min-width: 328px;
-$wdc-banner-height: 36px;
-$wdc-banner-mini-width: 210px;
+$wdc-banner-height: 40px;
+$wdc-banner-sticky-width: 222px;
 $wdc-banner-mobile-width: 240px;
-$wdc-banner-mobile-mini-width: 32px;
+$wdc-banner-mobile-sticky-width: 32px;
 $wdc-banner-padding: $wdc-spacing-xxs $wdc-spacing-s;
 $wdc-banner-bg-error: $wdc-color-error;
```

Comment:
Can you update `$wdc-color-error` in the core module to cinammon 500? Also the darker version should bump to 600. You can do it in another branch if you'd like since this one looks good to go 👍

Reply:
Actually, I'll just do this directly in the css-updates branch since it's very minor